enhancements/dns/coredns-egressfirewall-integration.md

```diff
@@ -385,32 +356,32 @@ N.A. This feature will go directly to Tech Preview.
 ### Upgrade / Downgrade Strategy
 
 Upgrade expectations:
-* On upgrade, the OVN-K master will create the corresponding ``EgressFirewallDNSName`` CRs for each DNS name in the
-existing EgressFirewall resources. The ``egressfirewall`` plugin will also start updating the ``.status``
-fields of the ``EgressFirewallDNSName`` CRs. The scenarios arising out of the order of the update of the various components
-are discussed in [Version Skew Strategy](#version-skew-strategy)
+* On upgrade, the OVN-K cluster manager will create the corresponding `EgressFirewallDNSName` CRs for each DNS name in the
```

Comment:
Perhaps an upgrade should delete any of the old CRs left if a downgrade was performed. Or is there a way that old CRs are marked as stale and cleaned up automatically?

Reply:
The ovn-k cluster manager will be responsible for the creation and deletion of the CRs. If any orphaned CR is found (for which there are no EgressFirewall DNS rules), then it will be deleted.